docs/datagrid/understanding-datagrid.md

```diff
@@ -10,29 +10,25 @@ Defaults
 Default options for all columns.
 
 ```js
-defaults: {
-    columns: {
-        editable: true
-    }
+columnDefaults: {
+    editable: true
 }
 ```
 
-For more information on `default` options see the [API reference]().
+For more information on `columnDefaults` options see the [API reference](https://api.highcharts.com/dashboards/#interfaces/DataGrid_Options.Options-1#columnDefaults).
 
 Caption
 ---------
 
 The caption of the datagrid grid.
 
 ```js
-settings: {
-    caption: {
-        text: 'Your title of datagrid'Heade
-    }
+caption: {
+    text: 'Title of the DataGrid'
 }
 ```
 
-For more information on `caption` options see the [API reference]().
+For more information on `caption` options see the [API reference](http://localhost:9005/dashboards/#interfaces/DataGrid_Options.Options-1#caption).
```

Comment:
It should not be a reference to localhost.

Reply:
oops, good catch :D